src/lib.rs

```diff
@@ -18,7 +18,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
-#![deny(warnings)]
+#![deny(clippy::needless_borrow)]
```

Comment:
@alex-semenyuk why removing `#![deny(warnings)]`?

Reply:
Saw this 
note: the lint level is defined here
  --> src/lib.rs:21:9
   |
21 | #![deny(warnings)]
   |         ^^^^^^^^
   = note: `#[deny(clippy::needless_borrow)]` implied by `#[deny(warnings)]`
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_borrow

https://github.com/objectionary/sodg/actions/runs/3233873480/jobs/5298203348#step:7:49